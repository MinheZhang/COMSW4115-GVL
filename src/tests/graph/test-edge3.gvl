int main() {
  node n1;
  n1 = create_node(0.0, 1.0, 1.0, 255, 255, 255, 0);
  node n2;
  n2 = create_node(2.0, 1.0, 1.0, 255, 255, 255, 0);
  edge e;
  e = create_edge(n1, n2, 1, 2, 3, 4);
  
  printi(get_edge_bold(e));
  printi(get_edge_r(e));
  printi(get_edge_g(e));
  printi(get_edge_b(e));

  node n3;
  n3 = get_edge_start(e);

  printf(get_node_x(n3));
  printf(get_node_y(n3));
  printf(get_node_radius(n3));
  printi(get_node_r(n3));
  printi(get_node_g(n3));
  printi(get_node_b(n3));

  node n4;
  n4 = get_edge_end(e);

  printf(get_node_x(n4));
  printf(get_node_y(n4));
  printf(get_node_radius(n4));
  printi(get_node_r(n4));
  printi(get_node_g(n4));
  printi(get_node_b(n4));

  return 0;
}