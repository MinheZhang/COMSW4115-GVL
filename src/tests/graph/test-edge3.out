1
2
3
4
0
1
1
255
255
255
2
1
1
255
255
255
